packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -560,6 +541,9 @@ abstract class PageTransitionsBuilder {
   /// const constructors so that they can be used in const expressions.
   const PageTransitionsBuilder();
 
+  /// Provideds a secondary transition to the previous route.
```

Comment:
These docs should be fleshed out. Maybe we need one place where we explain this whole thing very thoroughly and then link to it with a macro? Maybe including the example.

Reply:
Thanks, looks good now.